tests/foreman/ui_airgun/test_product.py

```diff
@@ -113,3 +114,54 @@ def test_positive_create_in_different_orgs(session, product_name):
                 product_name)[0]['Name'] == product_name
             product_values = session.product.read(product_name)
             assert product_values['details']['description'] == org.name
+
+
+@tier2
+def test_positive_product_create_with_create_sync_plan(session, module_org):
+    """Perform Sync Plan Create from Product Create Page
+
+    :id: 4a87b533-12b6-4d4e-8a99-4bb95efc4321
+
+    :expectedresults: Ensure sync get created and assigned to Product.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: medium
+    """
+    product_name = gen_string('alpha')
+    product_label = gen_string('alpha')
+    product_description = gen_string('alpha')
+    gpg_key = entities.GPGKey(
+        content=read_data_file(VALID_GPG_KEY_FILE),
+        organization=module_org
+    ).create()
+    plan_name = gen_string('alpha')
+    description = gen_string('alpha')
+    cron_expression = gen_choice(valid_cron_expressions())
+    with session:
+        # Create new product using different parameters
```

Comment:
That comment seems to be copied from other test and isn't relevant here, is it? Could you remove it?

Reply:
Yes, I will remove that. there has been changed requested for dependant PR, I will update PR with test result as well.